src/applications/static-pages/ivc-champva/10-7959f-2/App.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+const App = ({ formEnabled }) => {
+  if (formEnabled === undefined) {
+    return <va-loading-indicator message="Loading..." />;
+  }
+
+  if (formEnabled) {
+    return (
+      <>
+        <p>
+          You can register online, by mail, or by fax. You’ll need your Social
```

Comment:
Here, I would probably say "you can file a claim online....". CAIA will give us the final wording, but until we get it, we just want to be sure we differ from the registration.